### PR TITLE
WW 116:  swagger ui

### PR DESCRIPTION
**Pull request by HoXuanHieu**
add feature Api document, please check readme before using

### CODE REVIEW

File: src/main/java/com/example/codeE/CodeEApplication.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
 package com.example.codeE;
 
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
+import org.springframework.context.annotation.Configuration;
+//import springfox.documentation.swagger2.annotations.EnableSwagger2;
```
Comment (daovo739) [inline src/main/java/com/example/codeE/CodeEApplication.java:4]: please remove redundant code

Comment (daovo739) [inline src/main/java/com/example/codeE/CodeEApplication.java:4]: [M]: remove redundant code

**Review by daovo739 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -2,5 +2,4 @@
 backend code 
 
 # Swagger UI 
-To get swagger UI display, add '../swagger-ui.html' into the url 
-for example: https://localhost:8080/swagger-ui.html
\ No newline at end of file
+localhost: http://localhost:9000/swagger-ui/index.html#/
```
Comment (daovo739) [inline README.md:5]: [Q]: Is this match with server's port?

Comment (HoXuanHieu) [reply README.md]: any port that has been configured in application.propertities

**Review by Noro911 — APPROVED** [review]

File: src/main/java/com/example/codeE/CodeEApplication.java (hunk 1 above)
Comment (HoXuanHieu) [reply src/main/java/com/example/codeE/CodeEApplication.java]: has resolved

**Review by daovo739 — APPROVED** [review]

**Pull request merged**